Dev/src/components/StudentSelector/StudentSelectorModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import { View, StyleSheet, Modal, Text } from 'react-native'
+import PropTypes from 'prop-types'
+import TouchableText from 'components/TouchableText'
+import colors from 'config/colors'
+import fontStyles from 'config/fontStyles';
+import { screenHeight, screenWidth } from 'config/dimensions';
+import StudentSelectorComponent from './StudentSelectorComponent';
+
+//-------------------------------------------------------------
+// A Modal dialog that renders passed in data over ImageSelectionGrid
+// + a cancel button. It also styles the Modal to center it, wrap content,
+//  and adds shadow and other styling.
+//---------------------------------------------------------------
+export default class StudentSelectorModal extends Component {
+
+    render() {
+
+        cancelStyle = {...fontStyles.smallTextStyleDarkGrey,
+            alignSelf: 'center', 
+            marginTop: screenHeight * 0.015, 
+            marginBottom: screenHeight * 0.03 }
+        return (
+            <Modal
+                animationType="fade"
+                transparent={true}
+                presentationStyle="overFullScreen"
+                visible={this.props.visible}
+                onRequestClose={() => {
+                }}>
+
+                <View style={{ paddingBottom: 20 }}>
+                    <View style={styles.container}>
+                        <Text style={{...fontStyles.mainTextStyleDarkGrey, paddingLeft: 20, paddingTop: 15}}>Assign to:</Text>
```

Comment:
Should use screenHeight & width instead of exact value

Reply:
Good catch, yes.